app/controllers/pages_controller.rb

```diff
@@ -8,6 +8,10 @@ def about
     @title = "About Us"
   end
 
+  def calendar
+    @title = "Calendar"
```

Comment:
I added the @title variable to the view.  This is how it is done on the other pages that came with the theme, so I just kept it consistent with that.